catalog.html

```diff
@@ -42,52 +42,64 @@
   </header>
   <main class="content">
     <div class="banner">
-        <h2 class="banner__header">Explore catalog</h2>
+        <h1 class="banner__header">Explore catalog</h1>
         <ul class="banner__navigation">
             <li class="banner__nav-item"><a href="index.html" class="nav-link">Home</a></li>
             <li class="banner__nav-item"><a href="" class="nav-link card--nonactive">Catalog</a></li>
         </ul>
     </div>
     <section class="catalog__main-section">
         <form action="" method="post" class="left-block-filters">
-            <div class="clothing">
-                <h3 class="clothing__header">Clothing</h3>
-                <div class="clothing__options">
-                    <label class="options"><input type="checkbox" name="clothing" value="combatant">Combatant</label>
-                    <label class="options"><input type="checkbox" name="clothing" value="ninja">Ninja</label>
-                    <label class="options"><input type="checkbox" name="clothing" value="king">King</label>
-                    <label class="options"><input type="checkbox" name="clothing" value="queen">Queen</label>
-                    <label class="options"><input type="checkbox" name="clothing" value="robot">Robot</label>
+          <fieldset>
```

Comment:
Perfect 👍, but add the CLASS attribute please, and don't forget to change H3 tag to LEGEND tag

Reply:
Great, fixed